tests/resources/Nimbus-Util.robot

```diff
@@ -193,12 +187,14 @@ Deploy Nimbus vCenter Server Async
     [Return]  ${out}
 
 Deploy Nimbus Testbed
-    [Arguments]  ${user}  ${password}  ${testbed}
+    [Arguments]  ${user}  ${password}  ${spec}=  ${args}=
     Open Connection  %{NIMBUS_GW}
     Wait Until Keyword Succeeds  2 min  30 sec  Login  ${user}  ${password}
 
+    Put File  tests/resources/nimbus-testbeds/${spec}  destination=./%{BUILD_TAG}/testbeds/
+
     :FOR  ${IDX}  IN RANGE  1  5
-    \   ${out}=  Execute Command  ${NIMBUS_LOCATION} nimbus-testbeddeploy --lease 0.25 ${testbed}
+    \   ${out}=  Execute Command  ${NIMBUS_LOCATION} nimbus-testbeddeploy --lease 0.25 --testbedSpecRubyFile ./%{BUILD_TAG}/testbeds/${spec} ${args}
```

Comment:
Only include the spec arguments if a spec was provided.

Reply:
Instead I've included a default. This should be revisted based on https://github.com/vmware/vic/blob/e9345d6f0a0ff5c98caaf924538119f48ddbada4/tests/resources/Nimbus-Util.robot#L254 which doesn't provide a spec but does provide a testbedname - implies that a spec is not needed under some circumstances. It's unclear then why we are providing both a spec and a name in other locations - does this cause a merge of some kind?